src/traces/bar/attributes.js

```diff
@@ -35,8 +42,32 @@ module.exports = {
     y: scatterAttrs.y,
     y0: scatterAttrs.y0,
     dy: scatterAttrs.dy,
+
     text: scatterAttrs.text,
 
+    textposition: {
```

Comment:
> What would be the meaning of 'total' for inner bars? 

I'm thinking `'total'` would draw only one text item per bar group. But yeah, this would open up a cross-trace problem. Not so fun.